capellambse/model/__init__.py

```diff
@@ -324,3 +328,171 @@ def search(
     def by_uuid(self, uuid: str) -> common.GenericElement:
         """Search the entire model for an element with the given UUID."""
         return common.GenericElement.from_model(self, self._loader[uuid])
+
+    # pylint: disable=line-too-long
+    def update_diagram_cache(
+        self,
+        capella_cli: str,
+        image_format: t.Literal["bmp", "gif", "jpg", "png", "svg"] = "svg",
+        create_index: bool = False,
+        force: t.Literal["exe", "img"] | None = None,
+    ) -> None:
+        """Update the diagram cache if one has been specified.
+
+        If a diagram cache has been specified during the load of a
```

Comment:
The intention is to use the term "diagram cache" and not the argument name of that other method. At the same time I'm fine with using an argument name here. When referencing argument names they are normally italic (see https://github.com/python/cpython/blob/3.11/Doc/library/subprocess.rst).

What is the convention for the formatting of referenced argument names that are not arguments of the function/ method that is about to be documented?

I think one can write "during the load of". Yes, "while loading" sounds better, see 029f65a

Reply:
My intent was to establish a strong mental link to the `diagram_cache` c'tor argument, because ultimately that is what determines whether this method can work at all. IMO formatting it just like other function arguments helps with that. (I suppose we could make it a bit more explicit by writing something like "If a `diagram_cache` has been specified during `MelodyModel` construction [...]".)

Reply:
> IMO formatting it just like other function arguments helps with that.

So italic (here in reST `*diagram_cache*`) as it is usually done?

Reply:
Okay, accepted, see f565436